picard/util/tags.py

```diff
@@ -166,652 +227,1128 @@ def names(self, selector=None):
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'releaselanguage',
         shortdesc=N_('FIXME:releaselanguage'),
+        longdesc=N_('The language of the release as per ISO 639-3.'),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
-        'releasecountry',
-        shortdesc=N_('Release Country'),
+        'release_series',
+        shortdesc=N_('Release Series'),
+        longdesc=N_('A multi-value variable containing the series titles associated with the release.'),
+        is_hidden=True,
     ),
     TagVar(
-        'releasedate',
-        shortdesc=N_('Release Date'),
+        'release_seriescomment',
+        shortdesc=N_('Release Series Comment'),
+        longdesc=N_('A multi-value variable containing the series disambiguation comments associated with the release.'),
+        is_hidden=True,
+    ),
+    TagVar(
+        'release_seriesid',
+        shortdesc=N_('Release Series Id'),
+        longdesc=N_('A multi-value variable containing the series MusicBrainz Identifiers (MBIDs) associated with the release.'),
+        is_hidden=True,
+    ),
+    TagVar(
+        'release_seriesnumber',
+        shortdesc=N_('Release Series Number'),
+        longdesc=N_('A multi-value variable containing the series numbers associated with the release.'),
+        is_hidden=True,
     ),
     TagVar(
         'releasestatus',
         shortdesc=N_('Release Status'),
+        longdesc=N_(
+            'An indicator of the "official" status of the release. Possible values include "*official*", '
+            '"*promotional*", "*bootleg*", and "*pseudo-release*".'
+        ),
     ),
     TagVar(
         'releasetype',
         shortdesc=N_('Release Type'),
+        longdesc=N_(
+            'A multi-value tag containing the types of release assigned to the release group. See also '
+            '`%_primaryreleasetype%` and `%_secondaryreleasetype%`.'
+        ),
     ),
     TagVar(
         'remixer',
         shortdesc=N_('Remixer'),
+        longdesc=N_('The names of the remixer engineers associated with the track.'),
     ),
     TagVar(
         'replaygain_album_gain',
         shortdesc=N_('ReplayGain Album Gain'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'replaygain_album_peak',
         shortdesc=N_('ReplayGain Album Peak'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'replaygain_album_range',
         shortdesc=N_('ReplayGain Album Range'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'replaygain_reference_loudness',
         shortdesc=N_('ReplayGain Reference Loudness'),
+        longdesc=N_(''),
         is_calculated=True,
+        not_from_mb=True,
     ),
     TagVar(
         'replaygain_track_gain',
         shortdesc=N_('ReplayGain Track Gain'),
+        longdesc=N_(''),
```

Comment:
`longdesc` is optional, no need to set it to empty string, but I guess you added them in order to fill them.

Reply:
That was exactly it.  Before we merge I'll remove any blank ones.